src/QueueManager.php

```diff
@@ -171,10 +172,9 @@ public static function engine(string $name): SimpleClient
     /**
      * Push a single job onto the queue.
      *
-     * @param string|string[] $callable Either an array of [classname, method], or a string
-     *   to a statically callable function. When an array is used, the
-     *   class will be constructed by Queue\Processor and have the
-     *   named method invoked.
+     * @param string|string[] $callable The classname of a job that imlements the
```

Comment:
```suggestion
     * @param string|string[] $callable The classname of a job that implements the
```

Reply:
Typo is fixed. I also rebased to resolve conflicts.